### DISCUSSION TITLE
"Compatibility Issue: Sinytra Connector with Better End and Better Nether (Minecraft 1.21.1, NeoForge)"/"Проблема совместимости: Sinytra Connector с Better End и Better Nether (Minecraft 1.21.1, NeoForge)"

### DISCUSSION THREAD

**Discussion by sonle6wq** [Sinytra/Connector#1526] [Q&A] [answered]
Hello,

I'm not very familiar with GitHub, so it’s possible that this issue has been raised by someone else. I tried looking for information but, unfortunately, couldn’t find anything. In Discord, I was told that someone might have already reported this issue, but I wanted to write here to clarify.

I’ve encountered a compatibility issue with the Sinytra Connector mod on Minecraft 1.21.1 with NeoForge when used alongside the Better End and Better Nether mods.

With Better End, I noticed that I can't swim in horizontal directions. I can move up and down, but if I try to swim forward, backward, or in any other direction, the character gets stuck in place.
When using Better Nether, the game crashes on startup and shows an error message.
I’d appreciate any help or information on this issue.

Thank you!

Здравствуйте!

Я не слишком хорошо ориентируюсь в GitHub, так что, возможно, эта проблема уже была озвучена кем-то другим. Я пытался найти информацию, но, к сожалению, не смог. В Discord мне говорили, что кто-то уже сообщал об этой проблеме, но я решил написать сюда, чтобы уточнить.

Я столкнулся с проблемой совместимости мода Sinytra Connector на версии Minecraft 1.21.1 с NeoForge при использовании с модами Better End и Better Nether.

С модом Better End я заметил, что не могу плавать в горизонтальном направлении. Я могу двигаться вверх и вниз, но при попытке поплыть вперед, назад или в любом другом направлении персонаж застревает на месте.
При использовании Better Nether игра вылетает при запуске и выдает сообщение об ошибке.
Буду признателен за помощь или любую информацию по этому поводу.

Спасибо!

**Comment by Atrimilan** [comment, marked as answer]
For your swimming issue, someone has made a workaround : https://github.com/Sinytra/Connector/issues/1343#issuecomment-2466692376 _(open the link in the comment, and download the .jar file)_

With Better Nether, I have to restart the game once or twice and then it works 🤷‍♂️

**Reply by sonle6wq** [reply]
thanks, it worked helpd so much!

**Comment by AETTEA10** [comment]
I'm sorry I don't understand what you mean by restart the game, it never starts for me it just crashes on boot.

**Reply by Atrimilan** [reply]
BetterNether and swimming issues should have been fixed with version [2.0.0-beta.10+1.21.1](https://modrinth.com/mod/connector/version/2.0.0-beta.10+1.21.1), but I haven't tried it yet.
Note that you should NO LONGER use fix #1343 with this version ([see announcement in Discord](https://discord.com/channels/1141048834177388746/1142025823319240726/1422312278039990393)).

If you're still having crashes, send your log to the support channel, and maybe someone will be able to help you

**Reply by AETTEA10** [reply]
Sorry, I forgot to clarify I'm using the forge version 1.20.1 for my other mods. Is there a fix known for that version?

**Reply by Atrimilan** [reply]
Sorry I haven't use Sinytra in version 1.20.1, but if it's the same problem as it was in 1.21.1, then I don't think so 🫤
